docs/cli/cli-orders.md

```diff
@@ -303,7 +303,6 @@ of at least one being corrupted in creases
 Now we'll dive into the variety of ways to customize your order. These can all be
 combined with all the commands listed above.
 
-
```

Comment:
sorry for this and the extra space below - had to do a merge and didn't quite get it all right. I'll try to correct them, but if someone else can then go for it.